tools/lint.py

```diff
@@ -14,6 +14,7 @@ def main():
     os.chdir(root_path)
     eslint()
     pylint()
+    clippy()
```

Comment:
It'd be awesome if `tools/lint.py` supported running single linter (`--rs`), just like `tools/format.py`:
https://github.com/denoland/deno/blob/9a8d6fbd984d2c0f9775d90af64b3e95231dd21d/tools/format.py#L13-L33

Reply:
Agree, good idea!